docs/configuration-setup.rst

```diff
@@ -0,0 +1,193 @@
+
+.. code-block:: yaml
+
+   local-directory: $LOCAL_STORAGE
+
+
+No Local Storage
+----------------
+
+If your nodes do not have locally attached storage then we recommend that you
+turn off Dask's policy to write excess data to disk.  This must be done in a
+configuration file and must be separate from the ``jobqueue`` configuration
+section (though it is fine to include it in the same file).
+
+.. code-block:: yaml
+
+   jobqueue:
+     pbs:
+       cores: 36
+       memory: 100GB
+       ...
+
+   distributed:
+     worker:
+       memory:
+         target: False    # Avoid spilling to disk
+         spill: False     # Avoid spilling to disk
+         pause: .80       # Pause worker threads at 80% use
+         terminate: 0.95  # Restart workers at 95% use
+
+
+Network Interface
+-----------------
+
+HPC systems often have advanced networking hardware like Infiniband.
+Dask workers can take use of this network using TCP-over-Infiniband, this can
+yield improved bandwidth during data transfers.  To get this increased speed
+you often have to specify the network interface of your accelerated hardware.
+If you have sufficient permissions then you can find a list of all network
+interfaces using the ``ifconfig`` UNIX command
+
+.. code-block:: bash
+
+   $ ifconfig
+   lo          Link encap:Local Loopback                       # Localhost
+               inet addr:127.0.0.1  Mask:255.0.0.0
+               inet6 addr: ::1/128 Scope:Host
+   eth0        Link encap:Ethernet  HWaddr XX:XX:XX:XX:XX:XX   # Ethernet
+               inet addr:192.168.0.101
+               ...
+   ib0         Link encap:Infiniband                           # Fast InfiniBand
+               inet addr:172.42.0.101
+
+Alternatively, your IT administrators will have this information.
+
+
+Managing Configuration files
+----------------------------
+
+By default dask-jobqueue places a file at ``~/.config/dask/jobqueue.yaml`` with
+a commented out version of many different job schedulers.  You may want to do a
+few things to clean this up:
+
+1.  Remove all of the commented out portions that don't apply to you.  For
+    example if you use only PBS, then consider removing the entries under SGE,
+    SLURM, etc..
+2.  Feel free to rename the file or to include other configuration options in
+    the file for other parts of Dask.  The ``jobqueue.yaml`` filename is not
+    special, nor is it special that each component of Dask has its own
+    configuration file.  It is ok to combine or split up configuration files as
+    suits your group.
+3.  Ask your IT administrator to place a generic file in ``/etc/dask`` for
+    global use.  Dask will look first in ``/etc/dask`` and then in
+    ``~/.config/dask`` for any ``.yaml`` files preferring those in the user's
+    home directory to those in the ``/etc/dask``.  By providing a global file
+    IT should be able to provide sane settings for everyone on the same system
```

Comment:
Comment on this point: I've continued to get push back on this from our sys admins. This prompted this issue: https://github.com/dask/dask/issues/3798